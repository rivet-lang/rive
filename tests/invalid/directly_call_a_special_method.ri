struct Dropper {
    func __drop__(self) {
        _ = self;
    }
}

func main() {
    d := Dropper();
    d.__drop__();
}
