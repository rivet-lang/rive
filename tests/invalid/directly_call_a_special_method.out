tests/invalid/directly_call_a_special_method.ri:9:5: error: cannot call a special method directly
    9 |     d.__drop__();
      |     ^~~~~~~~~~~~
rivet: error: could not compile module `directly_call_a_special_method`, aborting due to previous error